proto/cacheclient.proto

```diff
@@ -1025,3 +1028,35 @@ message _SortedSetLengthByScoreResponse {
     _Missing missing = 2;
   }
 }
+
+message _SortedSetUnionStoreRequest {
+  enum AggregateFunction {
+    SUM = 0;
+    MIN = 1;
+    MAX = 2;
+  }
+
+  message _Source {
+    bytes set_name = 1;
+    optional float weight = 2
+    AggregateFunction aggregate = 3;
+  }
+
+  bytes set_name = 1;
+  repeated _Source sources = 2;
+  uint64 ttl_milliseconds = 3;
+  bool refresh_ttl = 4;
+}
+
+message _SortedSetUnionStoreResponse {
```

Comment:
I could make this simpler and return an uint for the number of elements in the resulting destination set. Which would be 0 if the result was empty and destination was not created.
Thoughts?

Reply:
I think this can just return the u64 yes

Reply:
All the other SortedSetLength* functions return uint32. It'd be inconsistent to change just this one to u64

Reply:
https://github.com/momentohq/client-protos/pull/310/commits/4d16a5dd8ce4a0cd9e6836374dd600d26d14442c

Reply:
🤷 unioning a few huge sorted sets maybe someday exceeds u32. I don't think the data type matters really, but if you like uint32 then go for it.

Reply:
You may have to call SortedSetLength on the resulting set also at some point and that returns u32 currently. I can make this u64 if we can live with the inconsistency

Reply:
ah, it doesn't matter much to me right now. I am slowly remembering that you might have some annoying things to deal with in javascript if you use u64 up front (around 2^53 due to ieee754 and js's _interesting_ choices, and dealing with string/number/bigint choices). u32 is fine.

Reply:
I believe uint32 -> uint64 is binary-compatible, so we can widen it in the future if necessary.

Reply:
I'll leave this as u32. We can do a wholescale widening in future.